### ISSUE TITLE
Could you release the checkpoint of reward models?

### ISSUE THREAD

**Issue by srzer** [Haoxiang-Wang/directional-preference-alignment#2]
Could you kindly release the checkpoint of the reward models trained for ultrafeedback and helpsteer? That would be great help to our evaluation. Thank you!

**Comment by Haoxiang-Wang** [comment]
Thanks for the interest -- I made some modification to the standard huggingface sequence classifier. Let me see how to push it to HuggingFace so users can easily run it. 

I will ping you here once it's done.

**Comment by Haoxiang-Wang** [comment]
Hi @srzer,

I just released the reward model at https://huggingface.co/Haoxiang-Wang/RewardModel-Mistral-7B-for-DPA-v1

It has 10-dimensional output, corresponding to the following attributes from HelpSteer and UltraFeedback 
`['helpsteer-helpfulness', 'helpsteer-correctness', 'helpsteer-coherence', 'helpsteer-complexity', 'helpsteer-verbosity', 'ultrafeedback-overall_score', "ultrafeedback-instruction_following", "ultrafeedback-truthfulness", "ultrafeedback-honesty",
                             "ultrafeedback-helpfulness"]`

Here is a sample code that you can try 
```python
from transformers import AutoModelForSequenceClassification,AutoTokenizer
import torch
device = 'cuda'
path = "Haoxiang-Wang/RewardModel-Mistral-7B-for-DPA-v1"
rm = AutoModelForSequenceClassification.from_pretrained(path, trust_remote_code=True).to(device)
tokenizer = AutoTokenizer.from_pretrained(path) 

input_template = "[INST] You must read the following conversation carefully and rate the assistant's response from score 0-100 in these aspects: helpfulness, correctness, coherence, honesty, complexity, verbosity\n\nUser: {prompt}\n\nAssistant: {response} [/INST]"

# Use a sample from HelpSteer validation set
prompt = 'What are some synonyms for the word "beautiful"?'
response = "Nicely, Beautifully, Handsome, Stunning, Wonderful, Gorgeous, Pretty, Stunning, Elegant"

model_inputs = tokenizer(input_template.format(prompt=prompt, response=response), return_tensors="pt").to(device)
with torch.no_grad():
    score = rm(**model_inputs).logits.squeeze().cpu().float().numpy()

print(score)
# [68.99269  69.62718  76.23071  33.48785  35.853596 63.833366 55.58917 68.7175 59.552124 46.465595]

# Convert from our scale (0-100) to HelpSteer scale (0-4) 
helpsteer_rewards_pred = (score[:5]-10)/20
print(helpsteer_rewards_pred)
# [2.9496346 2.981359  3.3115356 1.1743925 1.2926798]
# The actual rewards from the HelpSteer dataset for this sample are [3,3,4,2,2]
```


I will update the README later and add a demo jupyter notebook. Let me know if you encounter any issue when using this reward model!

**Comment by srzer** [comment]
Great! I will try it later
